nevergrad/functions/gym/gym_multi.py

```diff
@@ -227,18 +229,23 @@ def __init__(
         control: str = "conformant",
         neural_factor: int = 2,
         randomized: bool = True,
+        pb_index: int = -1,
     ) -> None:
         if os.name == "nt":
             raise ng.errors.UnsupportedExperiment("Windows is not supported")
         if "compilergym" in name:
             env = gym.make("llvm-ic-v0", observation_space="Autophase", reward_space="IrInstructionCountOz")
             self.uris = list(env.datasets["benchmark://cbench-v1"].benchmark_uris())
+            # For training, in the "stochastic" case.
+            self.csmith = list(env.datasets["generator://csmith-v0"].benchmark_uris())[:100]
```

Comment:
You're going to want to use `islice` here:

```suggestion
            from itertools import islice
            self.csmith = list(islice(env.datasets["generator://csmith-v0"].benchmark_uris(), 100))`
```

The full `benchmark_uris()` list is 2^32 elements long ;-)